server/graphql/document/src/queries/document.rs

```diff
@@ -0,0 +1,55 @@
+use async_graphql::*;
+use graphql_core::generic_filters::EqualFilterStringInput;
+use graphql_core::{standard_graphql_error::validate_auth, ContextExt};
+use repository::{DocumentFilter, EqualFilter};
+use service::auth::{Resource, ResourceAccessRequest};
+use service::document::document_service::{DocumentService, DocumentServiceTrait};
+use service::usize_to_u32;
+
+use crate::types::document::{DocumentConnector, DocumentNode};
+
+#[derive(Union)]
+pub enum DocumentResponse {
+    Response(DocumentConnector),
+}
+
+#[derive(InputObject, Clone)]
+pub struct DocumentFilterInput {
+    pub store_id: Option<EqualFilterStringInput>,
+    pub name: Option<EqualFilterStringInput>,
+}
+fn to_domain_filter(f: DocumentFilterInput) -> DocumentFilter {
+    DocumentFilter {
+        store_id: f.store_id.map(EqualFilter::from),
+        name: f.name.map(EqualFilter::from),
+    }
+}
+
+pub fn documents(
```

Comment:
I think front end would only really want to get the `latest document` or `document history`

Reply:
Yes this endpoint is for document_history (because I was playing with it in the frontend) so an endpoint that returns the latest document is needed.